robot/web/stream/server.go

```diff
@@ -412,3 +486,97 @@ func (server *Server) removeMissingStreams() {
 		utils.UncheckedError(streamState.Close())
 	}
 }
+
+// refreshVideoSources checks and initializes every possible video source that could be viewed from the robot.
+func (server *Server) refreshVideoSources() {
+	for _, name := range camera.NamesFromRobot(server.robot) {
+		cam, err := camera.FromRobot(server.robot, name)
+		if err != nil {
+			continue
+		}
+		existing, ok := server.videoSources[cam.Name().SDPTrackName()]
+		if ok {
+			existing.Swap(cam)
+			continue
+		}
+		newSwapper := gostream.NewHotSwappableVideoSource(cam)
+		server.videoSources[cam.Name().SDPTrackName()] = newSwapper
+	}
+}
+
+// refreshAudioSources checks and initializes every possible audio source that could be viewed from the robot.
+func (server *Server) refreshAudioSources() {
+	for _, name := range audioinput.NamesFromRobot(server.robot) {
+		input, err := audioinput.FromRobot(server.robot, name)
+		if err != nil {
+			continue
+		}
+		existing, ok := server.audioSources[input.Name().SDPTrackName()]
+		if ok {
+			existing.Swap(input)
+			continue
+		}
+		newSwapper := gostream.NewHotSwappableAudioSource(input)
+		server.audioSources[input.Name().SDPTrackName()] = newSwapper
+	}
+}
+
+func (server *Server) createStream(config gostream.StreamConfig, name string) (gostream.Stream, bool, error) {
+	stream, err := server.NewStream(config)
+	// Skip if stream is already registered, otherwise raise any other errors
+	var registeredError *StreamAlreadyRegisteredError
+	if errors.As(err, &registeredError) {
+		server.logger.Debugw("stream already registered", "name", name)
```

Comment:
```suggestion
		server.logger.Debugf("stream already registered", "name", name)
```
Or CDebugf

Reply:
Changed.

Why do we not like `Debugw`?

Reply:
`Debugw` is for error wrapping from my understanding, `Debugf` allows you to add a print statement to your logger line.